app/components/Layout/Header.tsx

```diff
@@ -237,6 +216,27 @@ const HeaderLayout = ({
           }
         `}
       </style>
+      <style jsx global>{`
```

Comment:
Instead of creating a global style, these should be moved to the `UserProfileDropdown`'s styles

Reply:
It's used in both components because Dashboard button is still in here. Is there another way to avoid copying all the styles?

Reply:
The `css` template method should allow you to have reusable css: https://github.com/vercel/styled-jsx#styles-outside-of-components

Reply:
We have an example of that in `containers/Forms/FormSharedStyles`

Reply:
Looks like I might be able to use resolve here since it's a scoped parent/child thing https://github.com/vercel/styled-jsx#the-resolve-tag